Classes/Network/FLEXNetworkRecorder.m

```diff
@@ -122,6 +123,9 @@ - (void)recordRequestWillBeSentWithRequestID:(NSString *)requestID request:(NSUR
 
         [self postNewTransactionNotificationWithTransaction:transaction];
     });
+
+    // generate the curl command for all request to debugger while networking debugging is enabled
+    NSLog(@"%@",[FLEXNetworkCurlLogger curlCommandString:request]);
```

Comment:
This will create a lot of log spew. Let's remove and just expose through the interface.